engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetFacesToCull.java

```diff
@@ -37,7 +37,9 @@
     private int mode;
 
     public SetFacesToCull(int mode) {
-        if (!modeNameMap.containsKey(mode)) {
+        if (mode != GL_BACK
```

Comment:
What happened here? Why did you revert to individual checks instead of checking against the map? 

In any case you should reverse the handling of this IF block. The current true case is the least likely to happen.